<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlyphEditor</class>
 <widget class="QWidget" name="GlyphEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="GLGlyphView" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>491</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Glyphstyle</string>
   </property>
   <widget class="QRadioButton" name="pies">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>106</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Pie Charts</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="geometry">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>106</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Geometry</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="projection">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>106</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Projection</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Primitives</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QRadioButton" name="points">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>106</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Points</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="vectors">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>106</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Vectors</string>
    </property>
   </widget>
  </widget>
  <widget class="QSlider" name="thresholdSlider">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>400</y>
     <width>19</width>
     <height>160</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="radiusSlider">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>400</y>
     <width>19</width>
     <height>160</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="normSlider">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>400</y>
     <width>19</width>
     <height>160</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="sizeSlider">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>400</y>
     <width>19</width>
     <height>160</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="thresholdBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>570</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="radiusBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>570</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="normBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>570</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="sizeBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>570</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>600</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Threshold</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>600</y>
     <width>59</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Radius</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>600</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Normalization</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>600</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Prim. Size</string>
   </property>
  </widget>
  <widget class="QComboBox" name="colorBox">
   <property name="geometry">
    <rect>
     <x>138</x>
     <y>660</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="displayBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>660</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="geoBox">
   <property name="geometry">
    <rect>
     <x>268</x>
     <y>660</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>690</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Display</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>690</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>298</x>
     <y>690</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Geometry</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>720</y>
     <width>92</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>720</y>
     <width>92</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>640</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Surfaces</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="updateBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>374</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>600</y>
     <width>80</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Min. Length</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="minlengthBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>570</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QSlider" name="minlengthSlider">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>400</y>
     <width>19</width>
     <height>160</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="minlengthPickBox">
   <property name="geometry">
    <rect>
     <x>398</x>
     <y>660</y>
     <width>120</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>414</x>
     <y>690</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Min. length</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLGlyphView</class>
   <extends>QWidget</extends>
   <header>glglyphview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
